toolkit/tools/pkg/imagecustomizerlib/testdata/base-config-malformed.yaml

```diff
@@ -0,0 +1,7 @@
+previewFeatures:
```

Comment:
What's malformed about this config file?

Reply:
Let's keep the `baseCfg.IsValid` in `buildConfigChainHelperValidation` for now. (We can revisit this decision in the future if it becomes a problem.)

But for specific individual fields (e.g. output path, input path), we can allow the check to happen against the merged config.